geoips/sector_utils/tc_tracks_database.py

```diff
@@ -240,7 +244,7 @@ def update_fields(tc_trackfilename, cc, conn, process=False):
     except IndexError:
         start_name = start_line[41]
 
-    if data == None:
+    if data is None:
         # print '    Adding '+tc_trackfilename+' to '+TC_DECKS_DB
         cc.execute(
             """insert into tc_trackfiles(
```

Comment:
Doesn't "is None" sometimes fail for like numpy arrays?  @jsolbrig could this potentially cause issues?

Reply:
flake8 wanted me to change that line to that. It was previously == None

Reply:
https://stackoverflow.com/questions/36783921/valueerror-when-checking-if-variable-is-none-or-numpy-array

It's "if not data" that doesn't work, not "if data is None".  This should be fine.